docs/api/README.md

```diff
@@ -0,0 +1,127 @@
+# Challenge/Response
+
+## Session setup
+
+* To start a new session and obtain a time-bounded nonce value:
+
+```
+>> Request:
+  POST /challenge-response/v1/newSession?nonceSize=32
+  Host: veraison.example
+  Accept: application/rats-challenge-response-session+json
+
+<< Response:
+  HTTP/1.1 201 Created
+  Content-Type: application/rats-challenge-response-session+json
+  Location: https://veraison.example/challenge-response/v1/session/1234567890
+
+  {
+    "nonce": "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0NTY3ODkwMTI=",
+    "expiry": "2030-10-12T07:20:50.52Z",
+    "accept": [
+      "application/psa-attestation-token"
+    ],
+    "state": "waiting"
+  }
+```
+
+## Asynchronous verification
+
+* Submit evidence for this session:
+
+```
+>> Request:
+  POST /challenge-response/v1/session/1234567890
+  Host: veraison.example
+  Accept: application/rats-challenge-response-session+json
+  Content-Type: application/psa-attestation-token
+
+  .....
+
+<< Response:
+  HTTP/1.1 202 Accepted
+  Content-format: application/rats-challenge-response-session+json
+
```

Comment:
This response seems to be lacking a Location to be used in the subsequent poll?

Reply:
Ah, wait - I didn't spot the sequence correctly: POST newSession and then POST to session as required. Might be worth adding a couple of lines about the interaction model to the top of the readme for anyone else reading this before a morning coffee.

Reply:
Cool, done in caca362.  The pic came out a bit bizarre though -- the UML client  puppet looks like UFO Robot Grendizer firing HTTP requests like photonics rays!